src/Notes/NavigationNudge.php

```diff
@@ -0,0 +1,101 @@
+<?php
+/**
+ * WooCommerce Admin: Navigation Nudge.
+ *
+ * Adds a note to nudge users to opt in to navigation.
+ */
+
+namespace Automattic\WooCommerce\Admin\Notes;
+
+defined( 'ABSPATH' ) || exit;
+
+use Automattic\WooCommerce\Admin\Features\Features;
+use Automattic\WooCommerce\Admin\Features\Navigation\Init as Navigation;
+
+/**
+ * Navigation Nudge.
+ */
+class NavigationNudge {
+	/**
+	 * Note traits.
+	 */
+	use NoteTraits;
+
+	/**
+	 * Name of the note for use in the database.
+	 */
+	const NOTE_NAME = 'wc-admin-navigation';
```

Comment:
What do you think of a slightly more specific name, `wc-admin-navigation-nudge` or something similar.

Reply:
Good call.